recipes/libgd/meta.yaml

```diff
@@ -14,7 +14,7 @@ requirements:
     - jpeg >=9* # use recent libjpeg
     - libpng
     - libtiff
-    - fontconfig # for libfontconfig.so
+    - fontconfig >=2.11 # for libfontconfig.so
     
   run:
     - libgcc # [not osx]
```

Comment:
You jjssed the `are configured for #` so hash is not used per default and as given [here in conda spec](http://conda.pydata.org/docs/building/meta-yaml.html#preprocessing-selectors) they are used in meta.yaml as comments 😎